packages/arb-avm-cpp/CMakeLists.txt

```diff
@@ -83,16 +91,58 @@ elseif (${AVM_FUZZER} MATCHES "honggfuzz")
     add_link_options(-fsanitize=address)
 endif ()
 
+if(NOT arb_node_LINKER_LIBS)
+    set(arb_node_LINKER_LIBS "")
+endif()
+
 find_package(Threads REQUIRED)
 find_package(GMP REQUIRED)
 find_package(GMPXX REQUIRED)
 
+set(ALT_MALLOC_FLAGS "-fno-builtin-malloc -fno-builtin-calloc -fno-builtin-realloc -fno-builtin-free")
+
+if(ENABLE_GPROF)
+    message(STATUS "Using GPROF")
+endif()
+
+if (ENABLE_GPROF)
+    find_package(Gperftools)
+    if(GPERFTOOLS_FOUND)
+        message(STATUS "Using GPROF")
+        include_directories(${GPERFTOOLS_INCLUDE_DIR})
+        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${ALT_MALLOC_FLAGS} -fno-omit-frame-pointer -g -pg")
```

Comment:
I think we don't need the `-fno-omit-frame-pointer` anymore since we always apply that globally

Reply:
:+1: